packages/dx-scheduler-core/src/plugins/month-view/computeds.js

```diff
@@ -1,10 +1,36 @@
 import moment from 'moment';
+import {
+  sortAppointments,
+  viewPredicate,
+  toPercentage,
+} from './../../utils';
+import {
+  findOverlappedAppointments,
+  adjustAppointments,
+  unwrapGroups,
+} from './../week-view/helpers';
+import {
+  sliceAppointmentByWeek,
+  getRectByDates,
+} from './helpers';
 
 const DAY_COUNT = 7;
 const WEEK_COUNT = 6;
+const MONTH_COUNT = 31;
 
-export const monthCells = (currentDate, firstDayOfWeek) => {
+export const endViewBoundary = (cells) => {
+  const cellsLastIndex = cells.length - 1;
+  const lastDate = moment(cells[cellsLastIndex][6].value);
+
+  return lastDate.startOf('day').add(1, 'days').toDate();
+};
+
+export const monthCells = (currentDate, firstDayOfWeek, intervalCount = 1) => {
   const currentMonth = moment(currentDate).month();
+  const currentMonths = [moment(currentDate).month()];
+  for (; currentMonths.length < intervalCount;) {
```

Comment:
This approach won't work if intervalCount is 3, for instance. It'll mark all cells from the last month.